content/en/docs/components/pipelines/user-guides/components/containerized-python-components.md

```diff
@@ -62,17 +62,18 @@ src/
 In this step, you'll provide `base_image` and `target_image` arguments to the `@dsl.component` decorator of your component in `src/my_component.py`:
 
 ```python
-@dsl.component(base_image='python:3.7',
+@dsl.component(base_image='python:3.11',
                target_image='gcr.io/my-project/my-component:v1')
 def add(a: int, b: int) -> int:
+    from math_utils import add_numbers
     return add_numbers(a, b)
 ```
 
 Setting `target_image` both (a) specifies the [tag][image-tag] for the image you'll build in Step 3, and (b) instructs KFP to run the decorated Python function in a container that uses the image with that tag.
 
 In a Containerized Python Component, `base_image` specifies the base image that KFP will use when building your new container image. Specifically, KFP uses the `base_image` argument for the [`FROM`][docker-from] instruction in the Dockerfile used to build your image.
 
-The previous example includes `base_image` for clarity, but this is not necessary as `base_image` will default to `'python:3.7'` if omitted.
+The previous example includes `base_image` for clarity, but this is not necessary as `base_image` will default to `'python:3.11'` if omitted.
```

Comment:
Same as above regarding Python 3.9.

Reply:
Same coment as above. Suppose you have a strong opinion I am fine with Python 3.9 or Python 3.11. What you decide it is fine for me.